Source/src/components/ComponentDirLight.cpp

```diff
@@ -41,28 +41,20 @@ float3 Hachiko::ComponentDirLight::GetDirection() const
     return transform->GetFront();
 }
 
-void Hachiko::ComponentDirLight::Save(JsonFormatterValue j_component) const
+void Hachiko::ComponentDirLight::Save(YAML::Node& node) const
 {
-    j_component["LightType"] = static_cast<int>(Type::DIRLIGHT);
-
-    const JsonFormatterValue j_color = j_component["Color"];
-    j_color[0] = color.x;
-    j_color[1] = color.y;
-    j_color[2] = color.z;
-    j_color[3] = color.w;
-
-    j_component["intensity"] = intensity;
-    j_component["active"] = active;
-    j_component["drawDirection"] = draw_direction;
+    node[LIGHT_TYPE] = static_cast<int>(Type::DIRLIGHT);
+    node[LIGHT_COLOR] = color;
+    node[LIGHT_INTENSITY] = intensity;
+    node[COMPONENT_ENABLED] = active;
```

Comment:
Should we add it in Load or remove from Save? I did not understand, sorry

Reply:
I guess that in the future it could be serialized for all components as commont behavior, for now its ok to not have it since i dont recall seing it in other component types

Reply:
I remember that I moved that setting to GameObject serialization but it was only in Save function. I added to Load and removed from the Lights components. So the feature is already done for the future use